src/debugAdapter/goDebug.ts

```diff
@@ -1363,6 +1363,16 @@ class GoDebugSession extends LoggingDebugSession {
 				Scope: scope,
 				Cfg: this.delve.loadConfig
 			};
+
+		// If coming from variables pane via the 'Copy Value' context menu action,
+		// first synchronously fetch the return value of the expression,
+		// and override the loadConfig to get the complete value in a second async request
+		evalSymbolArgs.Cfg = await this.evaluateRequestFetch(args, scope, this.delve.loadConfig).catch(err => {
```

Comment:
Consider the case where `evaluateRequestFetch` returns a config with no change i.e the current string length is well below the max limit configured by the user or the variable is not a string at all.
In these cases, we will be making 2 calls for no reason

Reply:
Done. I'm not really fond of the type guards, so if you have any improvement to suggest I'll be glad to apply it